soundfile.py

```diff
@@ -167,6 +167,33 @@
     __libsndfile_version__ = __libsndfile_version__[len('libsndfile-'):]
 
 
+class SoundFileError(RuntimeError):
+    """Base class for all SoundFile-specific errors.
+
+    :ivar code: libsndfile internal error number or `None`
+    :ivar error_string: raw libsndfile error message or `None`
+    """
+    def __init__(self, message, code=None, error_string=None):
+        self.args = (message, code, error_string)
+        self.message = message
+        self.code = code
+        self.error_string = error_string
+    def __str__(self):
+        return self.message
+
+
+def _sndfile_error(code, message=None, prefix=""):
+    """Return one of the SoundFileError exception, proper for the given
```

Comment:
"one of the": is this a remnant from earlier development?
Now there is only one exception class, right?

Reply:
yes, that is a lefover